crates/solidity/inputs/language/src/definition.rs

```diff
@@ -2579,9 +2579,26 @@ codegen_language_macros::compile!(Language(
                             name = EventParameter,
                             fields = (
                                 type_name = Required(TypeName),
-                                indexed_keyword = Optional(reference = IndexedKeyword),
+                                indexed_attribute = Optional(reference = EventIndexedAttribute),
                                 name = Optional(reference = Identifier)
                             )
+                        ),
+                        Struct(
+                            name = EventIndexedAttribute,
+                            fields = (
+                                indexed_keyword = Required(IndexedKeyword),
+                                // Replicate solc bug: https://github.com/ethereum/solidity/issues/13681
+                                repeated = Optional(
```

Comment:
I see. But the alternative in this PR also introduces a new `EventIndexedAttribute.repeated` field that is always there (in all versions), and Slang users would have to deal with it regardless. Not sure if that is a DX improvement over simply ignoring this bug/excluding the single contract from sanctuary. WDYT?

Reply:
I would love to ignore it myself but we discussed and agreed yesterday to include this and support this; it also turns out that this was *always* supported (from 0.4.11), so the range where this can occur is quite big.

In this case, the users are free to ignore the `repeated` field entirely, as it has no further semantic value, only the single indexed_keyword is what matters. We could help by adding more doc comments to the accessors for this field. The alternative *requires* knowledge about two variants (going through `variant`, picking a correct variant type for the child node) to be handled correctly, so I think that's a lot worse for what seems to be just a bug workaround.

Reply:
We could even have a suggestive name like `solc_bug_allow_repeated` or something like that to signal the users that they're not supposed to handle this and that it's unexpected.